feelpp/feel/feelmodels/modelmarkers.cpp

```diff
@@ -65,5 +65,14 @@ ModelMarkers::setPTree( pt::ptree const& p, ModelIndexes const& indexes )
     }
 }
 
+bool
+ModelMarkers::empty() const
+{
+    if( this->size() > 1 )
+        return false;
+    if( this->size() == 0 )
+        return true;
+    return this->begin()->empty();
```

Comment:
why overload this function (it's already in the parent class) ?

Reply:
if you have one marker equals to the empty string `""`, then the set is not empty but the markers should be empty.